client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/suggestion.js

```diff
@@ -38,7 +38,7 @@ const AddressSummary = ( { values, originalValues, countriesData, expandStateNam
 		const statesMap = ( expandStateName && ( countriesData[ country ] || {} ).states ) || {};
 		stateStr = statesMap[ state ] || state;
 	}
-	const countryStr = countriesData[ country ].name;
+	const countryStr = countriesData[ country ] ? countriesData[ country ].name : country;
```

Comment:
> When using the destination country Canada, I see 'CA' in the drop-down menu.

If the plugin doesn't contain the changes in https://github.com/Automattic/woocommerce-services/pull/1382, then this code can't know what `CA` is referring to, because the REST API only returns information about US and Puerto Rico. As I said in a previous comment, this is intended as a backwards-compatibility band-aid. We could get the countries information from another, non-WooCommerce-Services part of the Redux state, but I don't think it's worth it just for this.